src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/MarkDuplicatesSparkIntegrationTest.java

```diff
@@ -482,4 +482,15 @@ public void testAssertCorrectSortOrderMultipleBams() {
         args.addInput(new File(TEST_DATA_DIR,"example.chr1.1-1K.unmarkedDups.noDups.bam"));
         runCommandLine(args);
     }
+
+    @Test
+    public void testAssertCorrectSortOrderMultipleBamsOverriding() {
```

Comment:
add a test case for a single unsorted bam, and one that rejects when there's a coordinate sorted bam along side an unsorted bam